app/assets/v2/js/pages/onboard.js

```diff
@@ -229,14 +249,35 @@ $('.search-area input[type=text]').keypress(function(e) {
   }
 });
 
+const save_job_status = function() {
+  if (!$('.navbar #navbarDropdown').html()) {
+    _alert('No profile', 'error');
+  }
+  const job_search_status = $('#jobStatus').find(':selected').val();
+  const show_job_status = $('#showJobStatus').prop('checked');
+
+  const profile = {
+    url: '/api/v0.1/profile/' + $('.navbar #navbarDropdown').html().trim() + '/jobopportunity',
```

Comment:
Maybe we can use `document.contxt.github_handle` instead of check the string in the dropdown so if in the future the navbar change by design we are not going to have problems here.